src/routes/user/validation.js

```diff
@@ -19,3 +19,13 @@ export const loginValidation = (data) => {
 
   return schemaUser.validate(data);
 };
+
+export const editValidation = (data) => {
+  const schemaUser = Joi.object({
```

Comment:
To avoid code duplication you could compose register schema from login schema with the addition of missing fields like it was in the previous project in registration schema.